IdentityCore/src/network/request/MSIDAADRefreshTokenGrantRequest.m

```diff
@@ -23,6 +23,21 @@
 
 #import "MSIDAADRefreshTokenGrantRequest.h"
 #import "MSIDAADRequestConfigurator.h"
+#import "MSIDThumbprintCalculator.h"
+
+@interface MSIDAADRefreshTokenGrantRequest ()
+
+@property (nonatomic) NSMutableDictionary *thumbprintParameters;
+
+@end
+
+@interface MSIDRefreshTokenGrantRequest ()
```

Comment:
Based on our last discussion, we thought it would be best to have these request classes to conform to the common thumbprint calculator protocol that defines getFullRequestThumbprint and getStrictRequestThumbprint APIs. (read only protocols). Other than making this class conform to the new protocol, I did not make any other additional changes to this class, like its public/project/private settings, etc.. 

Otherwise, could you clarify on what you mean by having an internal header for this class?

Reply:
You can create a separate file called MSIDRefreshTokenGrantRequest+Internal and have those methods declared there.

Reply:
Aha! that makes sense. thanks for clarifying.

Reply:
addressed